bezier/src/lib.rs

```diff
@@ -81,6 +81,17 @@ pub type Point = euclid::Point2D<f32>;
 /// Alias for ```euclid::Point2D<f32>```.
 pub type Vec2 = euclid::Point2D<f32>;
 
+/// Alias for ```euclid::Size2D<f32>```.
+pub type Size = euclid::Size2D<f32>;
+
+/// Alias for ```euclid::Rect<f32>```
+pub type Rect = euclid::Rect<f32>;
+
+/// Allows to get a Rect structure which contains the Segment
+trait HasBoundingRect {
```

Comment:
Thinking of it, I've seen lots of "common methods" among bezier curves, so I'll remove the trait, because this is a big commitment indeed. :)